app/controllers/hub/notes_controller.rb

```diff
@@ -4,9 +4,11 @@ class NotesController < ApplicationController
     before_action :require_sign_in
     load_and_authorize_resource :client
     load_and_authorize_resource through: :client, only: [:create]
+    load_and_authorize_resource :user, parent: false, only: [:index]
     layout "admin"
 
     def index
+      @taggable_users = @users.to_json(only: [:name, :id], methods: :name_with_role_and_entity).to_s.html_safe
```

Comment:
This will be updated to limit to the appropriate set of users in a later PR. Based on performance, we also may need to request the list of users on the fly when tagging instead of loading them all at the beginning, but we'll see.